src/utils/users/checkUserPassword.js

```diff
@@ -0,0 +1,30 @@
+const { ApiError } = require('../../exeptions/auth.error');
+const { checkPassword } = require('../password/checkPassword');
+const { passwordSchema } = require('../validation/password');
+
+const checkUserPassword = async (
+  oldPass,
+  currentPass,
+  newPass,
+  confirmPass,
+) => {
+  if (!(oldPass && newPass && confirmPass)) {
+    throw ApiError.badRequest(
+      'Old,new and confirmed password is required for change password',
+    );
```

Comment:
The `ApiError` constructor arguments are incorrectly ordered. Ensure the correct order is `message, status, errors`.